schemas/yaml/Text.yml

```diff
@@ -1,5 +1,5 @@
 $schema: 'http://json-schema.org/draft-07/schema#'
-$id:     'http://schemas.digitallinguistics.io/Text-6.0.0.json'
+$id:     'http://schemas.digitallinguistics.io/Text-6.1.0.json'
```

Comment:
One thing to keep in mind with version bumps: We should adjust the version of a schema based on what's changed since the last release of the spec. For example, you only need to do a minor version bump on a schema if there hasn't already been a minor change for that schema since the last release. Otherwise, if we bump the version for each individual change, we'll wind up bumping 2-3 versions from one release to the next.

I'm not sure if that's relevant for the version bump here. I'll check when I do my review.

Reply:
That is good to know, I'm not sure if I was keeping that in mind!